libcipher/utils.py

```diff
@@ -0,0 +1,63 @@
+import string
+class EnglishChecker():
+    """This class serves the purpose of checking if a statement is English or not.
+       Two filter methods accomplish this by ensuring:
+        1. At least 85% of the message comprises of english alphabets
+        2. At least 25% of the words in the sentence are contained in
+           the dictionary file, an exhaustive list of english words
+    """
+    
+    def __init__(self):
+       
+        self.LETTERS_AND_SPACE = string.ascii_letters + string.whitespace
+        #f = open('/home/travis/build/Anupama-Github/libcipher/libcipher/dictionary.txt')
+        f = open('/home/travis/build/Chennaipy/libcipher/libcipher/dictionary.txt')
```

Comment:
Hardcoding paths in the library is *really* crazy. Pass the file object as argument to `__init__()` instead.